From 6a5dfd385ca86b9e40b1a73237408f76b671b490 Mon Sep 17 00:00:00 2001
From: Pauli Nieminen <suokkos@gmail.com>
Date: Mon, 09 Apr 2012 18:12:59 +0000
Subject: Don't complain about old kernel for 3.0+ kernels

Linux kernel version received bump to 3.0 that causes configure to
complain about older kernel. To avoid the warning configure needs to
check major and minor versions before micro.

Signed-off-by: Pauli Nieminen <suokkos@gmail.com>
---
diff --git a/configure.ac b/configure.ac
index 34fdc43..e4f6d5d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -83,7 +83,7 @@ Makefile
 
 AC_OUTPUT
 
-if [[ $KMICRO -lt 31 ]] ; then
+if [[ $KMAJOR -eq 2 -a $KMINOR -eq 6 -a $KMICRO -lt 31 ]] ; then
    echo
    echo "%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%@%"
    echo "@"  
--
cgit v0.9.0.2
